.github/CHANGELOG.md

```diff
@@ -1,32 +1,6 @@
-# Release 0.17.0-dev (development release)
+# Release 0.17.0 (current release)
```

Comment:
@antalszava I can't comment on it, but beware of accidental double-backticks :) I can see one down on line 44

![image](https://user-images.githubusercontent.com/2959003/128977718-c0b66708-9dc7-4469-bf40-d7817d375227.png)

Reply:
Updated! How come it's good to avoid them? :thinking: Would have thought both single and double-backticks work.

Reply:
Oh - Markdown only supports single backticks for code :) Double backticks rendering correctly is accident of some renderers, and is not always guaranteed to work.